src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -143,6 +147,42 @@ struct DisallowedParamsInserter {
   mobile_apis::VehicleDataResultCode::eType code_;
 };
 
+ResponseInfo::ResponseInfo()
```

Comment:
@dev-gh any way not clear. ResponseInfo is separate class and it has no access to AM if it won't be passed.